views/peeps/peep_list.erb

```diff
@@ -0,0 +1,6 @@
+<p>Peeps List</p>
+<ul>
+  <% @peeps.each do |message| %>
+    <li><%= message.peep %> <%= message.created_on[0...-10] %></li>
```

Comment:
I like you chose to display the more relevant part of the timestamp rather than the whole timestamp from database.